Add Estonian WinoGrande (#1165)
EuroEval/EuroEval: docs/datasets/estonian.md

the GPT4o model to translate the expected number of examples starting from the beginning of the respective splits.
The final dataset size is 1,024 / 256 / 1,767 for the training, validation and test splits, respectively.

Here are a few examples from the training split (note that unlike the test split these are machine translated):

```json
{"text": "Ian vabatahtlikult sõi Dennise menudo pärast seda, kui oli juba kausi söönud, sest _ põlgas soolte söömist.\n\na. Ian\nb. Dennis", "label": "b"}
```
```json
{"text": "Ian vabatahtlikult sõi Dennise menudo pärast seda, kui oli juba kausitäie söönud, sest _ nautis soolte söömist.\n\na. Ian\nb. Dennis", "label": "a"}
```
```json
{"text": "Ta ei tule kunagi minu koju, aga mina lähen alati tema majja, sest _ on väiksem.\n\na. kodu\nb. maja", "label": "a"}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 5
- Prefix prompt:
  ```
  Sulle esitatakse lüngaga (_) tekstülesanne ja kaks vastusevarianti (a ja b).
  ```
- Base prompt template:
  ```

>   Tekstülesanne: {text}
>   Vastus: {label}

Reviewer: I know that this is the template we use in the config, but to emphasise that choices are included, we "unpack" it a bit (we've done the same with all other multiple-choice datasets):

```suggestion
  Tekstülesanne: {text}
  Valikud:
  a. {option_a}
  b. {option_b}
  Vastus: {label}
```

I've taken the liberty of adding "Valikud:" ("Choices:", according to DeepL) here, to be consistent with the other multiple-choice datasets.

Author: Ah, is it only for documentation then?

Reviewer: Yep!

  ```
- Instruction-tuned prompt template:
  ```

>   Tekstülesanne: {text}
> 
> 
>   Sinu ülesanne on valida lünka sobiv vastusevariant. Vasta ainult {labels_str}. Muud vastused ei ole lubatud.

Reviewer: As with the prompt template, we include the choices here, including "Choices:":

```suggestion
  Tekstülesanne: {text}
  Valikud:
  a. {option_a}
  b. {option_b}
  
  Sinu ülesanne on valida lünka sobiv vastusevariant. Vasta ainult 'a' või 'b'. Muud vastused ei ole lubatud.
```

  ```

You can evaluate this dataset directly as follows:

```bash

> $ euroeval --model <model-id> --dataset winogrande_et

Reviewer: Dataset IDs are written in kebab case:

```suggestion
$ euroeval --model <model-id> --dataset winogrande-et
```

```

> ## Common-sense Reasoning

Reviewer: This is a duplicate of the section above
